website/static/css/custom.css

```diff
@@ -1,12 +1,20 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
+/*
```

Comment:
@KishaniKandasamy We'd better to keep the original MIT License announcement here.

Reply:
Fine then I will remove the ASF header and leave it as it is 👍 .

Reply:
👏